docs/_docs/langevin.md

```diff
@@ -0,0 +1,49 @@
+<script type="text/x-mathjax-config">
+MathJax.Hub.Config({
+  tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']]}
+});
+</script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
+
+# Deterministic and Stochastic Dynamics
+The possibility of conducting hybrid molecular dynamics - Monte Carlo (MDMC) schemes is possible by addition of the "Langevin move" in the `moves` section.
```

Comment:
- Rephrase. "possibility" and "possible" has similar meaning
- Give a yaml example
- Insert space after header.
- As we don't have anything else at the moment, perhaps just a single header "Langevin Dynamics"